Drift/Private/DriftFlexmatch.cpp

```diff
@@ -50,41 +56,72 @@ TStatId FDriftFlexmatch::GetStatId() const
 
 void FDriftFlexmatch::ReportLatencies()
 {
+	// Accumulate a mapping of regions->ping and once all results are in, PATCH drift-flexmatch
+	TSharedRef<TMap<FString, int>, ESPMode::ThreadSafe> LatenciesByRegion(new TMap<FString, int>());
```

Comment:
HTTP responses are sent back on the game thread, even if the request itself is threaded, so you probably don't need ThreadSafe here, unless I'm missing something

Reply:
I think this could be written as `auto LatenciesByRegion{ MakeShared<TMap<FString, int>>() };`